packages/api/src/controllers/v2/projects/mod.rs

```diff
@@ -63,6 +63,22 @@ impl DeliberationProjectController {
         } else {
             builder = builder.order_by_created_at_asc();
         }
+        if let Some(cond) = param.status {
+            let status = match cond.status {
+                ProjectStatusValue::Draft => DeliberationStatus::Draft,
+                ProjectStatusValue::Ready => DeliberationStatus::Ready,
+                ProjectStatusValue::InProgress => DeliberationStatus::InProgress,
+                ProjectStatusValue::Finish => DeliberationStatus::Finish,
+            };
+            match cond.op {
+                ProjectStatusOp::Equal => {
+                    builder = builder.status_equals(status);
+                }
+                ProjectStatusOp::NotEqual => {
+                    builder = builder.status_not_equals(status);
+                }
+            }
```

Comment:
```suggestion
            let status = cond.status.into();
            
            match cond.op {
                ProjectStatusOp::Equal => {
                    builder = builder.status_equals(status);
                }
                ProjectStatusOp::NotEqual => {
                    builder = builder.status_not_equals(status);
                }
            }
```
이런 형태가 될 수 있게 Into 를 구현하면 좋을거 같긴하네요.